docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -653,17 +653,17 @@ By default, this turns off any enabled coolant outputs. If called with the `R1`
 ; Usage
 M3.9 [Snnnnn] [Pnn] [Dnnn]
 
-; Start spindle assigned to current tool at 8000 RPM
+; Start spindle assigned to the current tool at 8000 RPM
 M3.9 S8000
 ```
 
 Wrapping the inbuilt `M3` command, `M3.9` starts the spindle in the clockwise direction at a particular speed and waits for it to accelerate to the target speed.
 
-If spindle feedback is configured, this command waits for a general purpose input to change state before returning.
+If spindle feedback is configured, this command waits for a general-purpose input to change state before returning.
 
-If spindle feedback is not configured, a static delay is used to make sure the spindle is up to speed before returning. The static speed used for this is timed by the operator during the wizard process, and modified based on the percentage speed-change being requested. So if the RPM change is 50% of the maximum RPM of the spindle, then the delay will be 50% of the operator-recorded delay.
+If spindle feedback is not configured, a static delay is used to make sure the spindle is up to speed before returning. The static speed used for this is timed by the operator during the wizard process, and modified based on the percentage speed change being requested. So if the RPM change is 50% of the maximum RPM of the spindle, then the delay will be 50% of the operator-recorded delay.
 
-Additionally, if expert mode is disabled, then this macro will pop up an operator confirmation / warning box when the spindle is going to accelerate from 0 rpm.
+Additionally, if Expert Mode is disabled, then this macro will pop up an operator confirmation/warning box when the spindle is going to accelerate from 0 rpm.
```

Comment:
```suggestion
Additionally, if Expert Mode is disabled, then this macro will pop up an operator confirmation when the spindle is going to accelerate from 0 rpm.
```
I'm not a fan of the no-space `/`, I think just having operator confirmation here is sufficient.